CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ option(ENABLE_IWYU "Enables include what you use program" FALSE)
 option(ENABLE_LTO "Enable LTO with compilation" TRUE)
 option(ENABLE_XRAY "Enable building with LLVM X-Ray" FALSE)
 set(USE_LINKER "" CACHE STRING "Allow overriding the linker path directly")
+option(ENABLE_ASAN "Enables Clang UBSAN" FALSE)
```

Comment:
```suggestion
option(ENABLE_UBSAN "Enables Clang UBSAN" FALSE)
```

Reply:
Thanks for catching this. It's so strange that even with this typo - `-DENABLE_UBSAN` works. :-/

Reply:
cmake is wacky like that. These option lines aren't strictly necessary and only really needed if you're wanting to use nice tools like `ccmake`.